src/main/java/hw6/ArrayValueCalculator.java

```diff
@@ -0,0 +1,38 @@
+package hw6;
+
+public class ArrayValueCalculator {
+
+    public static void main(String[] args) {
+        String[][] myArray = {
+                {"25", "35", "-48", "47"},
+                {"84", "37", "41", "14"},
+                {"25", "-69", "24", "18"},
+                {"84", "37", "25", "14"},
+//                {"84", "37", "ss", "14"},
+        };
+
+        try {
+            int sumAll = doCalc(myArray);
+            System.out.println("The sum of all array elements: " + sumAll);
+        }
+        catch (ArraySizeException | ArrayDataException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private static int doCalc(String[][] myArray) throws ArraySizeException, ArrayDataException {
+        int sum = 0;
+            for (int line = 0; line < myArray.length; line++)
+                for (int column = 0; column < myArray[line].length; column ++) {
+                    if (myArray.length != 4 || myArray[line].length != 4)
```

Comment:
In your solution possible to throw ArrayDataException for an invalid array size (ArraySizeException should be thrown because the array size is invalid, but we check before cells format). Example
String[][] arr = {
{"1", "XXX", "567", "1234"},
{"45"},
{"45", "998", "567", "1234"},
{"45", "998", "567", "456"}
};